app/src/pages/myNotes/DisplayNote.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./styles.module.css";
+
+/**
+ * DisplayNote component
+ * @constructor
+ */
+
+const DisplayNote = ({ selected }) => {
```

Comment:
We should name all the files noun-form like. Maybe it will be better to use `DisplayedNote` or `ActiveNote`?